Description: ignoring ui tests as they depends on rust-analyzer
Author: Eddie Billoir <lechatp@outlook.fr>
Index: bon/tests/integration/ui/mod.rs
===================================================================
--- bon.orig/tests/integration/ui/mod.rs
+++ bon/tests/integration/ui/mod.rs
@@ -1,5 +1,5 @@
-#[cfg(not(miri))]
-#[test]
+//#[cfg(not(miri))]
+//#[test]
 fn ui() {
     let t = trybuild::TestCases::new();
     t.compile_fail("tests/integration/ui/compile_fail/*.rs");
